docs/6.x/monitoring.md

```diff
@@ -180,60 +99,89 @@ metadata:
   name: smtp
 spec:
   host: smtp.host
-  port: <smtp port> # 465 by default
+  port: <smtp port>
   username: <username>
   password: <password>
----
+```
+
+Create the SMTP configuration:
+
+```bash
+$ gravity resource create smtp.yaml
+```
+
+The second resource is called `alerttarget`. It defines the alerts email recipient:
+
+```yaml
 kind: alerttarget
 version: v2
 metadata:
   name: email-alerts
 spec:
-  email: triage@example.com # Email address of the alert recipient
+   # email address of the alerts recipient
+  email: triage@example.com
 ```
 
-```bsh
-$ gravity resource create -f smtp.yaml
+Create the target:
+
+```bash
+$ gravity resource create -f target.yaml
```

Comment:
nit: `-f` will overwrite the existing configuration - should this be used as an example of introducing a resource? (Also it's confusing as `kubectl -f` specifies a filename, so I would use the full format: `gravity resource create --force target.yaml`).